src/extension/helper_process.c

```diff
@@ -146,6 +119,10 @@ dd_conn *nullable dd_helper_mgr_acquire_conn(client_init_func nonnull init_func)
         return NULL;
     }
 
+    zval runtime_path;
+    ZVAL_STR(&runtime_path, get_DD_APPSEC_HELPER_RUNTIME_PATH());
+    dd_on_runtime_path_update(NULL, &runtime_path);
```

Comment:
I don't really get why `dd_on_runtime_path_update` must be called here, where it's also a callback for the zai_config setting.

Reply:
Because updates only get called for changes, but not for the initial value.